tests/playwright/sanity/modules/v4-tests/inline-text-editing/canvas-editing.test.ts

```diff
@@ -115,4 +115,75 @@ test.describe( 'Inline Editing Canvas @v4-tests', () => {
 
 		expect( panelHTML ).toContain( NEW_CONTENT );
 	} );
+
+	test( "Style edited element differently than hello theme's default styles", async () => {
+		// Arrange
+		const containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		const headingId = await editor.addWidget( { widgetType: 'e-heading', container: containerId } );
+		let headingElement = editor.previewFrame.locator( `.elementor-element-${ headingId }` );
+
+		// Act
+		await editor.v4Panel.openTab( 'style' );
+		await editor.v4Panel.style.openSection( 'Typography' );
+		await editor.v4Panel.style.setFontWeight( 100 );
+		await headingElement.dblclick();
+
+		const inlineEditor = editor.previewFrame.locator( INLINE_EDITING_SELECTORS.canvasInlineEditor );
+
+		await inlineEditor.waitFor( { state: 'visible' } );
+
+		headingElement = inlineEditor.locator( `h2` );
+
+		// Assert
+		await expect.soft( headingElement ).toHaveCSS( 'font-weight', '100' );
+	} );
+
+	test( 'Global classes styles should render while editing', async () => {
+		// Arrange
+		const containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		const paragraphId = await editor.addWidget( { widgetType: 'e-paragraph', container: containerId } );
+		let paragraphElement = editor.previewFrame.locator( `.elementor-element-${ paragraphId }` );
+
+		// Act
+		await editor.v4Panel.openTab( 'style' );
+		await editor.v4Panel.style.addGlobalClass( 'hello' );
+		await editor.v4Panel.style.openSection( 'Typography' );
+		await editor.v4Panel.style.setFontSize( 100, 'px' );
+		await paragraphElement.dblclick();
+
+		const inlineEditor = editor.previewFrame.locator( INLINE_EDITING_SELECTORS.canvasInlineEditor );
+
+		await inlineEditor.waitFor( { state: 'visible' } );
+
+		paragraphElement = inlineEditor.locator( `p` );
+
+		// Assert
+		await expect.soft( paragraphElement ).toHaveCSS( 'font-size', '100px' );
+		await expect.soft( paragraphElement ).toHaveClass( /hello.*/ );
+		await editor.v4Panel.style.removeGlobalClass( 'hello' );
+	} );
+
+	test( 'Allow select all text by triple clicking when editor is rendered', async () => {
+		// Arrange
+		const containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		const headingId = await editor.addWidget( { widgetType: 'e-heading', container: containerId } );
+		const previewFrame = editor.getPreviewFrame();
+		let headingElement = previewFrame.locator( `.elementor-element-${ headingId }` );
+
+		// Act.
+		await headingElement.dblclick();
+
+		const inlineEditor = editor.previewFrame.locator( INLINE_EDITING_SELECTORS.canvasInlineEditor );
+
+		await inlineEditor.waitFor( { state: 'visible' } );
+		await page.waitForTimeout( 1000 );
```

Comment:
why wait for a magic number here? Can it not be waiting for some conditional like the line above it?